projects/core/ci_entrypoint/fournos.py

```
@@ -0,0 +1,162 @@
+        return None
+
+    try:
+        # Read and parse the FOURNOS config
+        with open(forge_config_path, 'r') as f:
+            forge_config = yaml.safe_load(f)
+
+        logger.info(f"Loaded FOURNOS config from {forge_config_path}")
+        logger.debug(f"Config content: {forge_config}")
+
+        # Create CI metadata directory
+        (artifact_path / CI_METADATA_DIRNAME).mkdir(parents=True, exist_ok=True)
```

Ruff (0.15.9)

[error] Undefined name `CI_METADATA_DIRNAME`

(F821)